includes/class-sensei-admin.php

```diff
@@ -1463,76 +1459,8 @@ public function theme_compatibility_notices() {
         if( isset( $_GET['sensei_hide_notice'] ) ) {
         	switch( esc_attr( $_GET['sensei_hide_notice'] ) ) {
 				case 'menu_settings': add_user_meta( get_current_user_id(), 'sensei_hide_menu_settings_notice', true ); break;
-				case 'theme_check': add_user_meta( get_current_user_id(), 'sensei_hide_theme_check_notice', true ); break;
```

Comment:
@donnapep I'm wondering if this function can be removed altogether? The only case being handled now is `menu_settings`, which AFAICT isn't being used anywhere. Both the URL argument (`menu_settings`) and the meta field (`sensei_hide_menu_settings_notice`) are not used anywhere else in the code.

For context, it looks like they were added [here](url), but the code to actually use them is gone.

Maybe this is beyond the scope of this PR, but it would be nice to clean up.

Reply:
I opted to leave it in because I wasn't sure if an extension might somehow be using it, and I'd rather be safe than sorry. 🙂 Otherwise I agree that it would be preferable to remove it.